bears/vcs/git/GitCommitBear.py

```diff
@@ -197,6 +202,34 @@ def check_body(self, body,
                                'HEAD commit. Please add one.')
             return
 
+        # Git keywords for closing issues
+        # https://help.github.com/articles/closing-issues-via-commit-messages/
+        git_keys = """close closes closed
+                      fix fixes fixed
+                      resolve resolves resolved""".split()
+
+        body_string = '\n'.join(body).lower()
+        if host_site != '' and any(key in body_string for key in git_keys):
+            urls = re.findall(r'(https?://\S+)', body_string)
+            good_url = False
+            for url in urls:
+                pieces = urlparse(url)
+                netloc = pieces.netloc
+                scheme = pieces.scheme
+                path = pieces.path
+                try:
+                    assert all([scheme, netloc, path])
+                except AssertionError:
+                    pass
+                if netloc.find(host_site) != -1 and path.find('issues') != -1:
```

Comment:
At least use `/issues/`.

But also please check that the part after `/issues/` is a number.